### DISCUSSION TITLE
Recent change of events, updates, moving forward

### DISCUSSION THREAD

**Discussion by Joabutt** [is-amazing/register#125] [Announcements]
Hey everyone, the creator of the is-really.cool here.

I have an apology to issue out to everyone about me being absent and gone for awhile, and the recent loss of domain.

Firstly, the reason why the project has been inactive for a really long time is because I have been extremely busy with school and new and exciting upcoming projects. School and pressure from peers has been taking up all of my time, leaving me with no time to maintain and update this project. I am extremely sorry for the people who have registered the domain waiting for a response but to no avail.

Secondly, the current status of the code is mediocre at least and was not written to be meant to last. I was hosting this on my own local raspberrypi and since then the raspberrypi has been crashing alot due to unsustainable code. I intend to find another more reliable hosting service and write better code.

Moving forward, now that I have a break in the school term, I intend to start a new project similar to this. I want to do a complete rehaul of the code and how the idea works, add new updates and functions. I firstly would like to obtain a new domain name, ending in .cool, I would love ideas on what it should be, please leave some comments below. As this is a pretty big project, I would appreciate help in accepting and merging the pull requests, if you want to apply as a maintainer, please dm me on discord @ joabutt. 

Thank you everyone and i hope to get the same continued support as i received on this project, as i embark on a new project and chapter in my development career, cheers!

**Comment by DogeDevReal** [comment]
Thanks, mate, Love you for doing all of this for us. True support from my side.
